cmake/FindFLAC.cmake

```diff
@@ -0,0 +1,66 @@
+# - Find FLAC
```

Comment:
No copyright header

Reply:
Thanks for catching this- it was copied from libsndfile and I have added the proper copyright at the top.